cli/azd/cmd/init.go

```diff
@@ -242,25 +250,27 @@ func (i *initAction) Run(ctx context.Context) (*actions.ActionResult, error) {
 			return nil, err
 		}
 
-		// Prompt to deploy to Azure
-		deploy, err := i.console.Confirm(ctx, input.ConsoleOptions{
-			Message:      "Deploy to Azure now?",
-			DefaultValue: true,
-		})
-		if err != nil {
-			return nil, err
-		}
-
-		if deploy {
-			// Call azd up
-			startTime := time.Now()
-			i.azd.SetArgs([]string{"up", "--cwd", azdCtx.ProjectDirectory()})
-			err := i.azd.ExecuteContext(ctx)
-			header = "New project initialized! Provision and deploy to Azure is completed in " +
-				ux.DurationAsText(since(startTime)) + "."
+		if i.flags.up {
+			// Prompt to deploy to Azure
+			deploy, err := i.console.Confirm(ctx, input.ConsoleOptions{
+				Message:      "Deploy to Azure now?",
```

Comment:
```suggestion
				Message:      "Do you want to run `azd up` now?",
```
I'd also add a help text with the content: 
"Template files have been initialized in your local directory. If you want to provision and deploy now without making changes, select Y. If not, select N." 

@vhvb1989 thoughts?